src/constants/coinLists.ts

```diff
@@ -91,7 +91,8 @@ import {
   NFTD,
   CLY,
   COOK,
-  SKILL
+  SKILL,
```

Comment:
We really shouldn't be pulling this data manually from another place.

We should have this data pulling from the `tokenlist` repo. I'll merge this PR, but this is some technical debt we'll need to fix in an upcoming release.

The reasoning is that we want to follow DRY:
DON"T REPEAT YOURSELF.

We have this coinlists already in tokenlists. Adding another place to maintain means extra work for people and more chance of things to break

Reply:
@HariSeldon23 we are now fetching tokens already from tokenlist. I have fixed that already in https://github.com/pangolindex/interface/pull/301